quiz-app/src/assets/translations/hin.json

```diff
@@ -0,0 +1,1017 @@
+					}
+				]
+			},
+			{
+				"id": 6,
+				"title": "पाठ 3 - सेंसर और एक्चुएटर्स के साथ भौतिक दुनिया के साथ बातचीत करें: पोस्ट-लेक्चर क्विज",
+				"quiz": [
+					{
+						"questionText": "डिजिटल सेंसर इस रूप में डेटा भेजते हैं:",
+						"answerOptions": [
+							{
+								"answerText": "वोल्टेज रेंजस",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "केवल हाई और लो वोल्टेज",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "ईमेल",
+								"isCorrect": "false"
+							}
+						]
+					},
+					{
+						"questionText": "बटन दबाने पर कौन सा डिजिटल सिग्नल भेजा जाता है?",
+						"answerOptions": [
+							{
+								"answerText": "0",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "1",
+								"isCorrect": "true"
+							}
+						]
+					},
+					{
+						"questionText": "आप पल्स-विड्त मॉड्यूलेशन का उपयोग करके डिजिटल डिवाइस से एनालॉग एक्ट्यूएटर को नियंत्रित कर सकते हैं",
+						"answerOptions": [
+							{
+								"answerText": "True",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "False",
+								"isCorrect": "false"
+							}
+						]
+					}
+				]
+			},
+			{
+				"id": 7,
+				"title": "पाठ 4 - अपने डिवाइस को इंटरनेट से कनेक्ट करें: प्री लेक्चर प्रश्नोत्तरी",
+				"quiz": [
+					{
+						"questionText": "IoT उपकरणों को हमेशा काम करने के लिए इंटरनेट से कनेक्ट होने की आवश्यकता होती है:",
+						"answerOptions": [
+							{
+								"answerText": "True",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "False",
+								"isCorrect": "true"
+							}
+						]
+					},
+					{
+						"questionText": "IoT डिवाइस हमेशा HTTP पर संचार करते हैं, वेब ऐप्स और अन्य वेब API के समान:",
+						"answerOptions": [
+							{
+								"answerText": "False",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "True",
+								"isCorrect": "false"
+							}
+						]
+					},
+					{
+						"questionText": "IoT डिवाइस अपने सभी निर्णय लेने के लिए क्लाउड पर निर्भर करते हैं:",
+						"answerOptions": [
+							{
+								"answerText": "False",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "True",
+								"isCorrect": "false"
+							}
+						]
+					}
+				]
+			},
+			{
+				"id": 8,
+				"title": "पाठ 4 - अपने डिवाइस को इंटरनेट से कनेक्ट करें: पोस्ट-लेक्चर प्रश्नोत्तरी",
+				"quiz": [
+					{
+						"questionText": "सेंसर से एकत्र किए गए डेटा और क्लाउड पर भेजे गए डेटा को कहा जाता है:",
+						"answerOptions": [
+							{
+								"answerText": "टेलीमेटरी",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "आदेश",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "मेजर्मेन्ट्स",
+								"isCorrect": "false"
+							}
+						]
+					},
+					{
+						"questionText": "IoT डिवाइस के ऑफ़लाइन होने पर कमांड का क्या होता है",
+						"answerOptions": [
+							{
+								"answerText": "यह हमेशा बुरा मानना चाहिए जब डिवाइस वापस ऑनलाइन है",
```

Comment:
"answerText": "यह हमेशा बुरा मानना चाहिए जब डिवाइस वापस ऑनलाइन है",

I am not really sure about its meaning. Kindly recheck